src/integrationTest/kotlin/com/sourcegraph/cody/edit/DocumentCodeTest.kt

```diff
@@ -139,6 +149,7 @@ class DocumentCodeTest : CodyIntegrationTextFixture() {
     assertNoActiveSession()
   }
 
+  @Test
   fun skip_testUndo() {
```

Comment:
`skip_` was a prefix to force the skip (like ignore annotation). Please remove it from the method name.